src/font/lv_font_loader.c

```diff
@@ -172,6 +176,21 @@ void lv_font_free(lv_font_t * font)
             if(NULL != dsc->glyph_dsc) {
                 lv_mem_free((void *)dsc->glyph_dsc);
             }
+#if LV_USE_FONT_DYNAMIC_LOAD
+            lv_font_fmt_txt_glyph_loader_t * loader =
+                (lv_font_fmt_txt_glyph_loader_t *)dsc->loader;
+
+            if(NULL != loader) {
+                if(NULL != loader->glyph_offset) {
+                    lv_mem_free((void *)loader->glyph_offset);
+                }
+                if(NULL != loader->fp) {
+                    lv_fs_close(loader->fp);
+                    lv_mem_free(loader->fp);
+                }
+                lv_mem_free(loader);
+            }
```

Comment:
Putting a LOG_ERR or LOG_WRN may help the user to track problems when a null happens here.

Reply:
> Putting a LOG_ERR or LOG_WRN may help the user to track problems when a null happens here.

Thanks, but this‘s release function, maybe it isn't necessary here.